docs/reference/commandline/service_create.md

```diff
@@ -474,6 +474,42 @@ accessible at the target port on every node regardless if there is a task for
 the service running on the node. For more information refer to
 [Use swarm mode routing mesh](https://docs.docker.com/engine/swarm/ingress/).
 
+### Templating
+
+Some flags of `service create` support the use of templating. Go's
```

Comment:
You can use templates for some flags of `service create`, using the syntax provided by the text/template package provided by the Go language.

Or something like that.